.env

```
@@ -27,4 +27,9 @@ JWT_SECRET=mySecretKey12345678901234567890123456789012345678901234567890
 JWT_EXPIRATION=86400000
 
 # Inter-service secret
-INTER_SERVICE_SECRET=inter-service-secret-key-123456789
\ No newline at end of file
+INTER_SERVICE_SECRET=inter-service-secret-key-123456789
+
+# Multisig Service Configuration
+DEPLOYER_PRIVATE_KEY=aba9dede35ae3042e09949a82605a33f77b1602a6cc463ee7b829d663accf570
+SERVICE_ACCOUNT_PRIVATE_KEY=aba9dede35ae3042e09949a82605a33f77b1602a6cc463ee7b829d663accf570
+OWNER_PRIVATE_KEYS=0xedd29206290aa5b7f1074d2e1819268fead031d6efba0b3f13bbc509ad5874a7,0x58b0aa9f91585c07a5bfcf20442d612a7b91ed625f270c8de34a926af615f7e0,0xd169c17d4fbaf02a119dbf293e71dbdecad72bc5853e6254be97b3ec63ec4d14,0xbacff54e83ec5a27c7100263eaa86f186d3225274d8c7b844623d193124e751a,0xc646ffde699b30aee56033d8128064c614f061fab6f107c0f9f424e4ff844645,0x068de925fa24b8aeeef24bbafd939db04f8759441aaaec6ff94638d238464007,0x80cc726119e3a4a97ad28da86d0216b59207306bb63d4f4a7a62d23fe874bf83,0xcbe91b911acffa8a6a8c2ff05ed825ec8ebc5e5acd730410fe7e9a2bff30254a,0xf8a435511948f1ed1edaffdd77c42be8af2abaec77bd04e52e59254954122f67
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)